tests/functional/disaster-recovery/regional-dr/test_acm_kubevirt_dr_integration.py

```
@@ -107,9 +114,26 @@ def test_acm_kubevirt_using_different_protection_types(
         )
-        assert navigate_using_fleet_virtualization(acm_obj)
         for i, vm in enumerate(cnv_workloads):
             standalone_flag = (not protection_type) or (i == 0)
+            if protection_type and i == 1:
+                # Wait for the standalone DRPC (created in iteration 0) to exist
+                # before opening the enrollment wizard in Shared mode. The ACM UI
+                # queries existing DRPCs when rendering the wizard; if the DRPC is
+                # not yet present the #shared-vm-protection radio button won't appear.
+                logger.info(
+                    f"Waiting for DRPC {protection_name}-drpc to exist"
+                    " before Shared enrollment"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)